packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/template.unsupported.tsx

```diff
@@ -1,26 +1,96 @@
+import { Button, Icon, Image, Text } from 'blockchain-info-components'
 import { FlyoutWrapper } from 'components/Flyout'
-import { Icon, Text } from 'blockchain-info-components'
-import { OwnProps } from '.'
+import { FormattedMessage } from 'react-intl'
+import {
+  LinkDispatchPropsType,
+  LinkStatePropsType,
+  OwnProps,
+  SuccessStateType
+} from '.'
 import React from 'react'
 import styled from 'styled-components'
 
-type Props = OwnProps
+export type Props = OwnProps &
+  SuccessStateType &
+  LinkDispatchPropsType &
+  LinkStatePropsType
 
 const Top = styled(FlyoutWrapper)`
   padding-bottom: 0px;
 `
 
+const CloseIcon = styled(Icon)`
+  position: absolute;
+  right: 44px;
+  top: 44px;
```

Comment:
Make the FlyoutWrapper position relative and make right: 0 and top: 0 instead of 44.